src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudServiceTest.kt

```diff
@@ -200,6 +201,59 @@ class CosmasGoogleCloudServiceTest {
         assertNotNull(stream.error)
     }
 
+    @Test
+    fun simpleAddPatch() {
+        createVersion("file", "1", "1")
+        addPatchToService("abc", "-", "1", 1, "1")
+        commit("1")
+    }
+
+    @Test
+    fun checkEmptyList() {
+        createVersion("file", "1", "1")
+        addPatchToService("abc", "-", "1", 1, "1")
+        commit("1")
+        createVersion("file1", "1", "1")
+        val list = this.service.getPatchList("1", "1")
+        assertEquals(0, list?.size)
```

Comment:
you are not sure if you get null or not? Test is essentially a specification, it says how code is expected to work. If you expect that `list` is null after this call, assert that it is null. If you expect that it is not null then assert that it is not null (you can use Kotlin's `!!` operator for that).

Reply:
ok